doc/source/cluster/kubernetes.rst

```diff
@@ -2,8 +2,24 @@
 
 .. _ray-k8s-deploy:
 
-Deploying on Kubernetes
-=======================
+The legacy Ray Kubernetes Operator
```

Comment:
Clearer wording here on the relative status of the two projects. This page is buried deeper than the new docs.